app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1041,58 +1041,60 @@ class BrowserTabFragment :
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
 
-        omnibar = Omnibar(
-            omnibarType = settingsDataStore.omnibarType,
-            binding = binding,
-            isUnifiedOmnibarEnabled = omnibarRepository.isUnifiedOmnibarLayoutEnabled,
-        )
-
-        webViewContainer = binding.webViewContainer
-        configureObservers()
-        viewModel.registerWebViewListener(webViewClient, webChromeClient)
-        configureWebView()
-        configureSwipeRefresh()
-        configureAutoComplete()
-        configureNewTab()
-        initPrivacyProtectionsPopup()
-        createPopupMenu()
-
-        configureNavigationBar()
-        configureOmnibar()
-
-        if (savedInstanceState == null) {
-            viewModel.onViewReady()
-            viewModel.setIsCustomTab(tabDisplayedInCustomTabScreen)
-            messageFromPreviousTab?.let {
-                processMessage(it)
+        lifecycleScope.launch {
+            omnibar = Omnibar(
+                omnibarType = settingsDataStore.omnibarType,
+                binding = binding,
+                isUnifiedOmnibarEnabled = omnibarRepository.isUnifiedOmnibarLayoutEnabled,
+            )
+            webViewContainer = binding.webViewContainer
+            viewModel.registerWebViewListener(webViewClient, webChromeClient)
+            configureWebView()
```

Comment:
nit: in here it calls `swipingTabsFeature.isEnabled` which might sometimes check the FF (first time). it's possible it's already cached at this point (`BrowserActivity` maybe) but if `configureWebView` is `suspend` you could also add a `withContext(dispatchers.io())` around `swipingTabsFeature.isEnabled` check; might help to prevent it appearing as a subtle issue later if the thing that is causing it to already be cached at this point is changed.